indico/modules/events/papers/templates/management/assignment.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'layout/full_content_base.html' %}
+{% from 'events/papers/management/_paper_assignment_list.html' import render_paper_assignment_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {% trans %}Paper Assignment{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {%- trans -%}
+        Assign <strong>judges</strong>, <strong>content reviewers</strong> and <strong>layout reviewers</strong>
+        to contributions in order to allow assessing its paper revisions.
+    {%- endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="toolbars space-after">
+        <div class="toolbar">
+            <div class="group">
```

Comment:
As explained in our [toolbars wiki page](https://github.com/indico/indico/wiki/TL-02:-Toolbars), single buttons in a toolbar don't need to be wrapped by the `.group` element anymore. Apply this to the rest of your templates as well! We currently have some (quite many actually) templates in Indico that still use the `.group` element even if it includes 1 button, but we'll remove them eventually. We just gotta make sure that any new code doesn't increase the amount of code we need to change!